src/Hare.java

```diff
@@ -0,0 +1,71 @@
+
+public class Hare {
+	private int x_location;
+	private int y_location;
+	private Directions direction;
+	
+	
+	
+	public Hare()
+	{
+		this.x_location =  0;
+		this.y_location = 0;
+		direction = Directions.randomDirection();
+	}
+	
+	public Hare(int x, int y)
+	{
+		this.x_location =  x;
+		this.y_location = y;
+		direction = Directions.randomDirection();
+	}
+	public void setLocation(int x,int y)
+	{
+		this.x_location =  x;
+		this.y_location = y;
+	}
+	public int getXLocation()
+	{
+		return this.x_location;
+	
+	}
+	public int getYLocation()
+	{
+		return this.y_location;
+	}
+	public void move(int units, Directions direction)
```

Comment:
Really both the Hare and the Hound should have a move() method that knows how that animal moves and moves it accordingly, rather than locating this logic in Manager. For the Hare, no context is required; for the Hound, it just needs to know where its target (the Hare) is located. It's okay to have this move() method that you wrote as well - but I would make it protected and put it in a superclass (then call it from the new move() method).